### PR TITLE
[FIX] l10n_br: invoice and refund accounts had been inverted on some taxes

### PR DESCRIPTION
**Pull request by oco-odoo**

### CODE REVIEW

**Comment by rvalyi** [comment]
cc @renatonlima 
please @cod-odoo check consistency with this PR https://github.com/odoo/odoo/pull/34160

**Comment by jco-odoo** [comment]
robodoo r+

**Comment by jco-odoo** [comment]
@kebeclibre

File: addons/l10n_br/data/account_tax_template_data.xml [hunk 1]
```diff
@@ -2309,7 +2309,7 @@
 	            (0,0, {
 	                'factor_percent': 100,
 	                'repartition_type': 'tax',
-	                'account_id': ref('account_template_101050505'),
+	                'account_id': ref('account_template_201010302'),
```
Comment (kebeclibre) [inline addons/l10n_br/data/account_tax_template_data.xml]: isn't it ref('...').id ?

Comment (oco-odoo) [reply addons/l10n_br/data/account_tax_template_data.xml]: Not in a .xml file ;)

**Comment by jco-odoo** [comment]
robodoo r+

**Pull request closed without merging**